data/remnant/remnant news.txt

```diff
@@ -0,0 +1,279 @@
+# Copyright (c) 2020 by Zitchas
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+
+
+phrase "00-07"
+	word
+		"00"
+		"01"
+		"03"
+		"04"
+		"05"
+		"06"
+		"07"
+
+
+
+phrase "00-07 days"
+	phrase
+		"00-07"
+	word
+		" days"
```

Comment:
Given that this phrase is just a previous phrase plus one word, it seems pretty pointless to me for it to be its own phrase unless you plan to use it elsewhere/more often. (It's especially redundant when the name of the phrase is just the phrase plus the word that it includes. This can be streamlined by using the following wherever you currently have `phrase "00-07 days"`:)
```
word
	"${00-07 days} days"
```
(This is functionally the exact same thing without having to define a new phrase elsewhere.)

Reply:
Nice and elegant. I didn't even know that was a thing for ES. I'll be using that, I think. (so long as it keeps the double digits)

Reply:
OK, I got it to work. For reference if anyone else ever digs through here, `${ }` basically acts like an inline way to call a phrase. So it isn't eliminating the base phrase, just the secondary phrase that combined the first phrase with the word "days".

Thanks!